### PR TITLE
BUGFIX: ONE-3630 Simplify documentation for login without sso

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

File: docs/sso.md [hunk 1]
```diff
@@ -49,11 +49,9 @@ This is how authentication process works:
 
         sdk.user.isLoggedIn().then((isLogged) => {
           if (isLogged) {
-            ReactDOM.render(
-             <App />,
-              document.getElementById('root')
-            );
+            // do some stuff, user is logged
```
Comment (evgeniabogranskaya) [inline docs/sso.md]: What do you mean by 'stuff'? =)

File: docs/sso.md [hunk 2]
```diff
@@ -49,11 +49,9 @@ This is how authentication process works:
...
           } else {
+            // redirect to login page, providing an url to redirect to upon successful login
```
Comment (evgeniabogranskaya) [inline docs/sso.md]: // redirect to the login page providing the URL to redirect to upon a successful login

File: docs/sso.md (hunk 1 above)
Comment (pbenes) [reply docs/sso.md]: means that you can do here whatever logged-in and authorized user can do, such as fetch md objects, execute reports etc. (primarily done via sdk.*.dosomething() helper methods)

Comment (evgeniabogranskaya) [reply docs/sso.md]: Gotcha, dekuji!

Comment (evgeniabogranskaya) [inline docs/sso.md]: How about:

the user is logged in, works in the app

?

Comment (pbenes) [reply docs/sso.md]: Anyway, it seems a good point to me, maybe we should rephrase somehow, what about having it simpler like
// user is logged here

Comment (evgeniabogranskaya) [reply docs/sso.md]: How about:

the user is logged in

?

**Comment by evgeniabogranskaya** [comment]
Proofread - LGTM

**Pull request merged**